examples/funloc/funloc_params.yml

```diff
@@ -44,16 +44,21 @@ preprocessing:
     tsss_dur: 60.
     st_correlation: 0.98
     trans_to: twa
+    cont_as_esss: True
   filtering:
     lp_cut: 50
     lp_trans: 10
     filter_length: auto
     fir_design: firwin
     phase: zero
   ssp:
+    cont_hp: 20
+    cont_hp_trans: 2
+    cont_lp: 40
+    cont_lp_trans: 2
     proj_nums: {
-      subj_01: [[2, 2, 0], [1, 1, 3], [0, 0, 0], [1, 1, 0], [1, 1, 0]],
```

Comment:
Before noticing these lists I was warming up to the idea of `yaml` file(s). I assume if the `param` is not listed in the `yaml` I can proceed as before? @larsoner

Reply:
My experience is that @larsoner has been good about setting reasonable `params `defaults for all fields left undefined in the yaml file.  The yaml approach makes the code cleaner and (for me) a way to archive previous mnefun pipeline runs.

Reply:
My only concern with `{k:v}` where `v = list([],[])` is that the yaml can get quite unruly for a run on 18-20 subjects.

Reply:
I see.  I haven't used the double lists on proj_nums myself, though my last run had 35 subjects.  Just keep track of those commas, I guess!

Reply:
Actually the [MNE-BIDS API ](https://mne.tools/mne-bids/stable/generated/mne_bids.mark_bad_channels.html?highlight=bad%20channels) is built to comb through lists so this a useful approach for downstream integration. WRT to ACQ bad channels I think we should always point the to the `prebad.txt`, there is really no need to duplicate the contents of that SLF anywhere else.

Reply:
Yes YAML just overrides what are defaults in `Params`, which are certainly not optimal defaults but should be backward compatible.

Regarding bads personally if I were starting a study from scratch I wouldn't use any `_prebad.txt` anymore but would instead put everything in the YAML. It makes the definition longer but it's fewer files to commit and keep track of.